custom/data.ts

```diff
@@ -1,73 +1,72 @@
-import { faker } from "@faker-js/faker";
```

Comment:
[via ChatGPT review bot]

I've noticed you removed the faker library and in every place you were using it, you gave a fixed value. If the purpose is to give more real data here, that's cool. But be aware that having dynamic data is useful when developing interfaces because they can show us how our application behaves when dealing with unexpected or large inputs.

You did a great job using JavaScript ES6 template literals for the `text` under `description`. Always remember to separate concerns, it might be a good idea to move the text content into a separate file or internationalization layers like i18n. This way we can keep our code cleaner and easier to manage.

Good improvement on the addition of your real GitHub links, showing potential employers your real projects is always a good idea. Even better when we are in a portfolio project.

For more information on data in client-side code, visit this link: [Do not use real data for development](https://security.stackexchange.com/questions/41078/is-it-a-bad-idea-to-use-an-api-key-in-client-side-code-web-or-mobile)}

For more information on i18n, visit this [i18n and JavaScript](https://www.devexpress.com/Support/Center/Question/Details/A2984/localization-how-to-correctly-use-it-with-javascript)